sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/db/migration/V8520__Attachments.java

```diff
@@ -0,0 +1,173 @@
+package uk.ac.bbsrc.tgac.miso.db.migration;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.logging.Logger;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import uk.ac.bbsrc.tgac.miso.core.util.LimsUtils;
+
+public class V8520__Attachments implements JdbcMigration {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00101)